standalone/README.md

```diff
@@ -19,36 +19,135 @@ In order to build and use the PSENscan Standalone C++ Library you need Ubuntu ve
+  ```
+  cd psen_scan_v2/standalone/
+  mkdir build
+  ```
+
+- Lastly, execute the following:
+  ```
+  cd build/ && cmake .. && make
+  ```
 
 The library should now be built and available under `psen_scan_v2/standalone/build/libpsen_scan_v2_standalone.a`
 
-## Usage example
+### Usage example
 An example application, which prints distance data to the screen, is built by default and can be executed in the `build` folder:
 ```
 ./psen_scan_v2_standalone_app
 ```
 
+## Get Started on Windows
+### Dependencies
+- Visual Studio
+- Boost (https://www.boost.org/)
+- console_bridge https://github.com/ros/console_bridge
+- fmt (https://github.com/fmtlib/fmt)
+
+### Build and install dependencies
+#### Boost
+
+- First download source from https://www.boost.org/
+- Unzip the file and open the folder e.g. `boost_1_75_0`
+- Open a command line and run
+  ```
+  bootstrap.bat
+  ```
+- Build boost with
+  ```
+  b2 -j8 toolset=msvc-14.1 address-model=64 architecture=x86 link=static threading=multi runtime-link=shared --build-type=complete stage
+  ```
+  `toolset` is the compiler you want to use, here 14.1->MSVC2017 <br>
+  `address-model` build for 64 or 32 bit <br>
+  `link` has to be static, because cmake expects them so
+
+Boost should now be successfully built.
+
+#### console_bridge
+- Download source from Github (https://github.com/ros/console_bridge)
+- Open `Visual Studio Developer Console`, the version you want
+- Open the `console_bridge`-folder, with the `CMakeLists.txt` in it
+- Generate Visual Studio Project
+  ```
+  cmake -G "Visual Studio 15 2017 Win64" .
+  ```
+  _With `cmake -G` you can check which generators are abailable_
+
+- Build the generated project with 
+  ```
+  cmake --build . --target ALL_BUILD
+  ```
+  `--target` is the available Visual Studio Project we generated before, you can look in the folder how they are named
+
+- Install console_bridge with 
+  ```
+  cmake --build . --target INSTALL
+  ```
+  _You may need admin privileges_
+
+#### fmt
+- Download source from Github (https://github.com/fmtlib/fmt)
+- Open `Visual Studio Developer Console`, the version you want
+- Open the `fmt`-folder, with the `CMakeLists.txt` in it
+- Generate Visual Studio Project: 
+   ```
+   cmake -G "Visual Studio 15 2017 Win64" .
+   ```
+   _With `cmake -G` you can check which generators are abailable_
+- Build the generated project with 
+  ```
+  cmake --build . --config Release --target ALL_BUILD
+  ```
+  `--target` is the available Visual Studio Project we generated before, you can look in the folder how they are named
+- Install with 
+  ```
+  cmake --build . --config Release --target INSTALL
+  ```
+  _You may need admin privileges_
+
+### Build psen_scan_v2_standalone
+- Download source from Github
+- Open `Visual Studio Developer Console`, the version you want
+- Open the `standalone`-folder, with the `CMakeLists.txt` in it
+- Create a build folder and open it 
+  ```
+  mkdir build64 & pushd build64
```

Comment:
Why is a build folder necessary for `psen_scan_v2` but not for `console_bridge` and `fmt`? Should we keep that consistent?

Reply:
Probably not strictly necessary but since this is the folder you might want to work in it doesn't hurt to add it here.